libs/Onboarding/src/Onboarding/NewAccountController.cpp

```diff
@@ -11,16 +11,8 @@ namespace Status::Onboarding
 
 namespace StatusGo = Status::StatusGo;
 
-NewAccountController::NewAccountController(std::shared_ptr<AccountsServiceInterface> accountsService, QObject *parent)
-    // TODO: remove dev dev setup after the final implementation
-    : m_name("TestAccount")
-    , m_nameIsValid(true)
-    , m_password("1234567890")
-    , m_passwordIsValid(true)
-    , m_confirmationPassword("1234567890")
-    , m_confirmationPasswordIsValid(true)
-    // END dev setup
-    , m_accountsService(accountsService)
+NewAccountController::NewAccountController(std::shared_ptr<AccountsServiceInterface> accountsService, QObject* parent)
```

Comment:
Can you please create alias for shared pointers:
`
using AccountsServiceInterfacePtr = std::shared_ptr<AccountsServiceInterface>;
`

Reply:
I usually don't do it if I want to forward declare them. I updated the implementation and used the alias only in the `*.cpp` files but still forward declared in the header. Do you think the compile time gained is not worth lowering readability?